app/graphql/mutations/authentication/sign_up.rb

```diff
@@ -1,10 +1,6 @@
 module Mutations
   module Authentication
     class SignUp < BaseMutation
-      argument :name, String, required: true
```

Comment:
@ManuSG8 I was thinking about the sign_up, and I strongly think that reduce facilitate the sign_up process to the maximum is a good idea. Later, we could obligate the user, in the first login, to enter some other data (if needed).

Probably from the coach's perspective, for basic coaching, the `email` is more than enough information.

---

Please, when you read this, let me know what's your opinion on this! We can discuss further and do the modifications we see as convenient

Reply:
I agree, let's continue this way